spec/unit/models/services/service_operation_shared.rb

```diff
@@ -206,7 +233,7 @@ def update_operation(type, state)
   describe '#update_succeeded?' do
     context 'when there is no operation' do
       it 'returns true' do
```

Comment:
Very small thing: The description needs to be adapted, i.e. `it 'returns false' do`

Reply:
done